bin/chef-zero

```diff
@@ -55,6 +55,11 @@ OptionParser.new do |opts|
     options[:log_file] = value
   end
 
+  opts.on("--enterprise", "Whether to run in enterprise mode") do |value|
```

Comment:
It's really a separate question. chef-client -z doesn't support full-on enterprise mode yet (there are separate PRs for that).